scripts/amd/gemm/matmul.py

```diff
@@ -284,20 +333,45 @@ def parse_args():
     parser.add_argument("-k", type=int, default=0)
     parser.add_argument("-dtype", type=str, default='fp16', help="Input data type, default is fp16")
     parser.add_argument("--specify_type", action='store_true', default=False, help="Whether user specify data type, default false")
-    parser.add_argument("--specify_size", action='store_true', default=False, help="Whether user specify input matrix size, default false")
+    parser.add_argument("--use_size_file", action='store_true', default=False, help="Whether user specify input matrix size")
     parser.add_argument("--compare", action='store_true', default=False, help="Whether check result correctness")
-    parser.add_argument("--gemm_size_file", type=str, default="", help='yaml file to indicate matrix size')
+    parser.add_argument("-gemm_size_file", type=str, help='yaml file to indicate matrix size')
```

Comment:
what is the difference between - and -- ?

Reply:
`-` means there will be a value for this parameter. `--` is just a flag for turning on/off, no value for this parameter.